index.html

```diff
@@ -52,10 +53,21 @@ <h2 id="to-the-map-h2">
 <div id="leafletDiv"></div>
 
 <script src="./index.ts"></script>
-<script src="vendor/Leaflet.AccuratePosition.js"></script>
+        <script src="vendor/Leaflet.AccuratePosition.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
```

Comment:
Why the extra import of jquery? It is already available via NPM.

It can be used with `import $ from "jquery` in the typescript file